go.mod

```diff
@@ -3,34 +3,37 @@ module github.com/libp2p/go-libp2p-core
 go 1.17
 
 require (
-	github.com/btcsuite/btcd v0.20.1-beta
+	github.com/btcsuite/btcd/btcec/v2 v2.1.3
```

Comment:
Can we only update the btcec dependency in this PR please?

Reply:
Yes, will do

Reply:
When I reverted the go.mod and go .sum changes, and then made the change to v2, ran go mod tidy it updated a lot of versions to the semantically compatible updated version. Are you suggesting I manually maintain the files instead of using the tools? Or is there a way of using the tools to only make very specific changes? (I'm reading the manuals, but if there's a pointer I'm happy to follow it)

Reply:
I can't reproduce this.
```
go get github.com/btcsuite/btcd/btcec/v2
go mod tidy
```
works perfectly fine.

Reply:
I'm still seeing a bit more change than I expect following the same steps.

Reply:
This looks fine now. Thanks.